### ISSUE TITLE
Implement an easy way to attach keyboard shortcuts

### ISSUE THREAD

**Issue by radubrehar** [infinite-table/infinite-react#227]
We need an easy way to allow people attach keyboard shortcuts and perform actions when a shortcut is pressed.

**Comment by radubrehar** [comment]
We'll add `keyboardShortcuts` prop, which is an array of objects of the following shape:

```ts
{
  key: 'cmd+shift+x',
  when?: (context) => true/false,
  handler: (context, event) => ...
}
```

**Comment by radubrehar** [comment]
Shipped in 3.3.3
